.solr_wrapper.yml

```diff
@@ -0,0 +1,5 @@
+# Place any default configuration for solr_wrapper here
```

Comment:
This file now gives the project 2 solr_wrappers i don't know which one is actually getting referenced here, or if either of them get referenced.  I suspect they can both be deleted. 2nd one is in config/ should clean it up for clarity while we're doing this.